developer_manual/core/theming.rst

```diff
@@ -1,166 +1,239 @@
 Theming ownCloud
 ================
-Themes can be used to customize the look and feel of ownCloud.
-Themes can relate to the following topics of owncloud:
 
-* Theming the web-frontend
-* Theming the owncloud Desktop client
+Themes can be used to customize the look and feel of ownCloud.
+Themes can relate to the following topics of ownCloud:
 
-This documentation contains only the Web-frontend adaptations so far.
+* The web front-end
+* The ownCloud Desktop client
 
+This documentation contains only the web front-end adaptations so far.
 
-Getting started
+Getting Started
 ===============
-A good idea getting starting with a dynamically created website is to inspect it with **web developer tools**, that are found in almost any browser. They show the generated HTML and the CSS Code, that the client/browser is receiving:
-With this facts you can easily determine, where the following object-related attributes for the phenomenons are settled:
 
-* place
-* colour
-* links
-* graphics
+A good idea, when getting started with a dynamically created website, is to inspect it with the web developer tools available in the major web browsers, which include: `Google Chrome`_, `Mozilla Firefox`_, and `Safari`_. 
+Among a host of other information, the developer tools show the generated HTML and the CSS code that the client (browser) is receiving.
+With this information you can easily determine all you need to know about the following theme details:
 
-The next thing you should do, before starting any changes is:
-Make a backup of your current theme(s) e.g.:
+* Place
+* Color
+* Links
+* Graphics
 
-* cd …/owncloud/themes
-* cp -r example mytheme
+The next thing you should do, before starting any changes, is to make a backup of your current theme(s). 
+Here's an example of how to do so::
 
+  cd path/to/your/owncloud/themes
+  cp -r example mytheme
 
-Creating and activating a new theme
+Creating and Activating a New Theme
 ===================================
-There are two basic ways of creating new themings:
 
-* Doing all new from scratch
-* Starting from an existing theme or the example theme and doing everything step by step and more experimentally
+There are two basic ways of creating new themes:
+
+1. Create a new one from scratch.
+2. Start from an existing theme and do everything step-by-step.
 
-Depending on how you created your new theme it will be necessary to
+Perhaps the simplest way is to start with the sample theme. 
+This is located in the ``themes`` folder of your ownCloud install, inside the folder called ``example``. 
```

Comment:
@phisch Better name it "theme-example" or "theme-mycompany" to have all themes at the same location between all apps?